modules/op-release-notes-1-9.adoc

```diff
@@ -433,4 +433,18 @@ With this update, {pipelines-title} General Availability (GA) 1.9.2 is available
 [id="fixed-issues-1-9-2_{context}"]
 == Fixed issues
 
-* Before this update, an OLM misconfiguration issue had been introduced in the previous version of the release, which prevented the upgrade of {product-title}. With this update, this misconfiguration issue has been fixed.
\ No newline at end of file
+* Before this update, an OLM misconfiguration issue had been introduced in the previous version of the release, which prevented the upgrade of {product-title}. With this update, this misconfiguration issue has been fixed.
+
+[id="release-notes-1-9-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.9.3
+
+With this update, {pipelines-title} General Availability (GA) 1.9.3 is available on {product-title} 4.10 in addition to 4.11, 4.12, and 4.13.
+
+[id="fixed-issues-1-9-3_{context}"]
+== Fixed issues
+
+* This update fixes the performance issues for huge pipelines. Now, the CPU usage is reduced by 61% and the memory usage is reduced by 44%.
+
+* Before this update, a pipeline run would fail if a task did not run because of its `when` expressions. This update fixes the issue by preventing the validation of a skipped task result in pipeline results. Now, the pipeline result is not emitted and the pipeline run does not fail because of a missing result.
+
+* This update fixes the `pipelineref.bundle` conversion to the bundle resolver. Now, the conversion feature sets the value of the `kind` field to `Pipeline`.
```

Comment:
`This update fixes the `pipelineref.bundle` conversion to the bundle resolver for v1beta1 API,`

Reply:
Done!